lib/iex/endpoints/historial_prices.rb

```diff
@@ -0,0 +1,28 @@
+module IEX
+  module Endpoints
+    module HistoricalPrices
+      def historical_prices(symbol, options = {})
+        if options[:range] == 'date'
+          raise ArgumentError if options[:date].nil?
```

Comment:
`unless options[...]` feels cleaner

Reply:
Personally I prefer the `if` route but happy to change.